posthog/models/feature_flag/feature_flag.py

```diff
@@ -297,6 +297,17 @@ def get_cohort_ids(
 
         return list(cohort_ids)
 
+    def scheduled_changes_dispatcher(self, payload):
+        if "field" not in payload or "value" not in payload:
+            raise Exception("Invalid payload")
+        elif payload["field"] == "active":
```

Comment:
all these updates should go through the serializer, as otherwise validation will not happen correctly.

For example, the case I mentioned earlier around disallowing group changes in scheduled changes

Reply:
This also needs to handle backwards compatibility so just accessing `self.filters` can lead to incorrect updates, which the serializer should automatically handle